blueprints/data-solutions/data-platform-minimal/variables.tf

```diff
@@ -23,6 +23,9 @@ variable "composer_config" {
       pypi_packages            = optional(map(string), {})
       env_variables            = optional(map(string), {})
       image_version            = optional(string, "composer-2-airflow-2")
+      cloud_data_lineage_integration = optional(object({
+        enabled = bool
+      }), { enabled = true })
```

Comment:
this data type is really weird. Would it be enough to use something simpler like `cloud_data_lineage_integration = optional(bool, false)`?

Reply:
I did in this way to replicate the type in the underline resource.